### DISCUSSION TITLE
Is there a package for installing the .NET Hosting Bundle?

### DISCUSSION THREAD

**Discussion by mvarblow** [microsoft/winget-pkgs#34490] [Q&A] [answered]
I see packages for the .NET runtime and SDK, also for the ASP.NET and Desktop runtimes. I even see packages for the just released VS 2022 SKUs. But I don't see a package for the .NET Hosting Bundle. This seems like a surprising oversight. Am I just missing it?

https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/hosting-bundle

Thanks!

**Comment by jedieaston** [comment, marked as answer]
Total oversight. Oops.

#34494 will resolve it, you should be able to do `winget install dotnet-hosting-bundle` once it clears.

**Reply by mvarblow** [reply]
Wow, that's great. Thanks for the immediate response! I see that you added 5.0.12. Can you also add the new 6.0.0 LTS hosting bundle?

https://dotnet.microsoft.com/download/dotnet/6.0

Thanks, again!

**Reply by jedieaston** [reply]
.NET 6 is now available from the same package!

**Reply by mvarblow** [reply]
Thank you!!
